CIFAR/models.swift

```diff
@@ -46,49 +46,44 @@ public struct KerasModel : Layer {
     var dense1: Dense<Float>
     var dropout3: Dropout<Float>
     var dense2: Dense<Float>
-    public init(learningPhaseIndicator: LearningPhaseIndicator) {
+    public init(learningPhase: LearningPhase) {
```

Comment:
You can remove this parameter completely.

```suggestion
    public init() {
```